.github/workflows/tests.yml

```diff
@@ -142,12 +142,10 @@ jobs:
           'cpu-clang')
             export CC=clang
             export CXX=clang++
-            DDC_CMAKE_CXX_FLAGS="-Werror=extra-semi -Werror=old-style-cast"
           ;;
           'cpu-gcc')
             export CC=gcc
             export CXX=g++
-            DDC_CMAKE_CXX_FLAGS="-Werror=extra-semi -Werror=old-style-cast"
```

Comment:
I temporarely remove the `-Werror=extra-semi old-style-cast"` to fix the CI while we discuss the best solution to have a KokkosKernel installation with @yasahi-hpc functions available.

Reply:
@tpadioleau  wants us to update CI to pre-install KK to avoid this error. For me, it is OK as a temporary solution

Reply:
Ok I do it or you do ?

Reply:
If you can do that, it would be great. I would like to work on KK side for further fixes, more functionalities (teams), etc..

Reply:
Okay if you can just take a look at the CI, I think the error comes from a missing [[maybe_unused]] in your code.

Reply:
Sure. I will fix that after tbsv is merged to KK. Or you would like a fix as soon as possible?

Reply:
No it's not urgent at all

Reply:
@yasahi-hpc can you take a look at the CI ? Thomas made a commit to install KK but there is a problem related to:
https://github.com/yasahi-hpc/kokkos-kernels/blob/6a9535d8200054484fc4158bebb5d8673a6318cf/batched/dense/impl/KokkosBatched_Gbtrs_Serial_Impl.hpp#L21

Maybe remove the `../src/` ?

Reply:
True. 
I will make a fix soon